docs/guides/8-continuous-integration.md

```diff
@@ -47,6 +47,30 @@ Learn more about [CircleCI Configuration](https://circleci.com/docs/2.0/config-i
 
 Spectral has a pre-built [Spectral GitHub Action](https://github.com/stoplightio/spectral-action) which should speed up implementing Spectral in your GitHub repository.
 
+## GitLab
+
+GitLab users can add the following to their `.gitlab-ci.yml` files:
+
+```yaml
+stages:
+  - lint
+
+lint:spectral:
+  stage: lint
+  image:
+    name: stoplight/spectral
+    entrypoint: [""]
+  script:
+    - spectral lint -D -f junit -o spectral-report.xml openapi.yaml
+  artifacts:
+    when: always
+    reports:
+      junit: $CI_PROJECT_DIR/spectral-report.xml
+      expire_in: 2 weeks
+```
+
+Note that this CI job exposes Spectral results on the merge request page, along with any other test output you may have.
```

Comment:
Could you please make a small update here to tell users to update the `openapi.yaml` part with the path to their API description?

Something like what we have for CircleCI, could be: 

"Make sure to change `openapi.yaml` to point to whatever documents you want to lint. Note that this CI..."